pyproject.toml

```diff
@@ -10,7 +10,7 @@ python = "^3.8,<3.11"
 defusedxml = "^0.7.1"
 
 [tool.poetry.group.dev.dependencies]
-demisto-sdk = "1.22.0"
+demisto-sdk = {git = "https://github.com/demisto/demisto-sdk.git", rev = "master"}
```

Comment:
reminder to revert

Reply:
will be reverted after finishing up all testings